casper/src/test/scala/coop/rchain/casper/util/rholang/RuntimeManagerTest.scala

```diff
@@ -31,6 +32,18 @@ class RuntimeManagerTest extends FlatSpec with Matchers {
     result.status.isFailed should be(true)
   }
 
+  it should "capture rholang parsing errors and charge for parsing" in {
```

Comment:
Please add a test for a success scenario as well, asserting that the deducted cost is `costOfParsing + costOfExecution`. The latter can be reliably obtained by running the reducer.